chapters/cmdbuffers.txt

```diff
@@ -761,14 +761,6 @@ time.
 include::{generated}/validity/structs/VkCommandBufferInheritanceInfo.txt[]
 --
 
-If ename:VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT was not set when
-creating a command buffer, that command buffer must: not be submitted to a
-queue whilst it is already in the <<commandbuffers-lifecycle, pending
-state>>.
-If ename:VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT is not set on a
-secondary command buffer, that command buffer must: not be used more than
-once in a given primary command buffer.
-
 [NOTE]
 .Note
 ====
```

Comment:
BTW, not quite sure what this note is saying about some "patching"...

Reply:
@krOoze Patching refers to the driver modifying the data and executable code recorded into the command buffer at queue-submission time or during execution.

Reply:
@polarina Sorry, I know what patching means. What I do not understand is is why it is brought up here with relation to `VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT`. One would assume the copy is the real performance problem. Whether one patches the original or the copy as part of the submission, that part is the same, no?

Reply:
@krOoze Correct, the copy is the performance problem. It is the use of `VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT` that forces that copy, the implementation can not patch in-place which it could otherwise do without that bit set.

Reply:
oh, right, the patching is a write access that necessitates the copy in the first place.